### PR TITLE
chore: create apache monitoring quickstart

### PR DESCRIPTION
**Pull request by aliandersonsn**
Search Nurture's recommendations for the description section of Apache quickstart.

## Summary

A concise description of the changes being introduced.
If this is a new Observability Pack, please provide a description of its use cases and a list of the Pack's components.

### Screenshots

Images of any visual changes, such as a dashboard.

### CODE REVIEW

File: packs/apache/config.yml [hunk 1]
```diff
@@ -5,8 +5,35 @@ name: Apache
 
 # Description of the pack
 description: |+
-  Official New Relic pack for Apache.
-  Use this pack together with the New Relic Apache On Host Integration to get insight into the performance of your Apache instances.
+  # Complete Apache Monitoring Kit
```
Comment (jpvajda) [inline packs/apache/config.yml:8]: this feels a bit wordy to me from a developer POV, so I'd like to ask to try to reduce the amount of text and provide a more succinct descrption, cc @bradleycamacho for his opinion

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: packs/apache/config.yml [hunk 2]
```diff
@@ -7,33 +7,19 @@ name: Apache
 description: |+
   # Complete Apache Monitoring Kit
```
Comment (jpvajda) [inline packs/apache/config.yml]:
```suggestion
  ## Complete Apache Monitoring Kit
```

**Review by jpvajda — DISMISSED** [review]
just corrected the

File: packs/apache/config.yml (hunk 2 above)
Comment (jpvajda) [reply packs/apache/config.yml]: @bradleycamacho if you can accept this suggestion, this looks good!

**Comment by jpvajda** [comment]
fixed merge conflict.

File: packs/apache/config.yml (hunk 2 above)
Comment (bradleycamacho) [reply packs/apache/config.yml]: I couldn't accept this suggestion since it was outdated, but I pushed up a commit implementing it.

File: packs/apache/config.yml [hunk 3]
```diff
@@ -5,8 +5,35 @@ name: Apache
```
[the remark this answers is not in the document]
Comment (bradleycamacho) [reply packs/apache/config.yml:8]: Looks like Ali is going to open some PRs with the summary separately. Let's go ahead and merge this in now if we can!

**Review by polfliet — APPROVED** [review]
